kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -59,15 +63,19 @@
     private static final String CLASS_DELETE_COMPUTER_PROBLEM_FACT_CHANGE = "org.kie.server.testing.DeleteComputerProblemFactChange";
     private static final String CLASS_CLOUD_GENERATOR = "org.kie.server.testing.CloudBalancingGenerator";
 
+    private static final long EXTENDED_TIMEOUT = 300000;
```

Comment:
Is that 300_000 milliseconds? Just know that since java 7 you can use underscores in numeric literals to improve readability.

Reply:
@jhrcek Right, it is 300 000 milliseconds (5 minutes). Will adjust the readability. The default defined in Kie server client is 5 or 10 seconds. It is raised to 1 minute in Kie server tests.